### ISSUE TITLE
[Questions] Is requestedSessionTimeout fixed value in createSession()?

### ISSUE THREAD

**Issue by ChaJiWon** [open62541/open62541#2291]
Description
===========
Hello :-)
I wonder something about createSession API.
I saw that value of requestedSessionTimeout is fixed to **1200000** like below code.

Does it have any reason about this fixed variable? 
Is it related with OPC-UA Specification? or Is it Policy of open62541?

If I want to increase requestedSessionTimeout, Should I modify this variable in open62541?

```
static UA_StatusCode
createSession(UA_Client *client) {
    UA_CreateSessionRequest request;
    UA_CreateSessionRequest_init(&request);

    request.requestHeader.timestamp = UA_DateTime_now();
    request.requestHeader.timeoutHint = 10000;
    UA_ByteString_copy(&client->channel.localNonce, &request.clientNonce);
    request.requestedSessionTimeout = 1200000;
    request.maxResponseMessageSize = UA_INT32_MAX;
    UA_String_copy(&client->endpointUrl, &request.endpointUrl);

    UA_CreateSessionResponse response;
    __UA_Client_Service(client, &request, &UA_TYPES[UA_TYPES_CREATESESSIONREQUEST],
                        &response, &UA_TYPES[UA_TYPES_CREATESESSIONRESPONSE]);

    UA_NodeId_copy(&response.authenticationToken, &client->authenticationToken);

    UA_StatusCode retval = response.responseHeader.serviceResult;
    UA_CreateSessionRequest_deleteMembers(&request);
    UA_CreateSessionResponse_deleteMembers(&response);
    return retval;
}
```

Background Information / Reproduction Steps
===========================================

Checklist
=========
Please provide the following information:

 - [ O ] open62541 Version (release number or git tag): 0.2 version
 - [ X ] Other OPC UA SDKs used (client or server): 
 - [ O] Operating system: 
 - [ X ] Logs (with `UA_LOGLEVEL` set as low as necessary) attached
 - [ X ] Wireshark network dump attached
 - [ X ] Self-contained code example attached
 - [ X ] Critical issue

**Comment by jpfr** [comment]
This is something we should make configurable in the `ClientConfiguration` structure.

**Comment by ChaJiWon** [comment]
@jpfr,
Oh, "1200000" isn't intended value?
Then I think it can be made like configurable of secureChannelLifeTime.

**Comment by jpfr** [comment]
Sure the number is intended.
But it is not mandated to be exactly that in the standard.

**Comment by Pro** [comment]
Fixed with #2338
